code_detection/tests/helpers.py

```diff
@@ -21,13 +21,13 @@ def test(text: str, expected_result: str) -> None:
 
         if actual_result != expected_result:
             print(
-                f"TEST #{test_counter} FAILED in {execution_time_ms:.4f} milliseconds",
+                f"  TEST #{test_counter} FAILED in {execution_time_ms:.4f} milliseconds",
             )
             print(f"    - Expected: {expected_result!r}")
             print(f"    -   Actual: {actual_result!r}")
         else:
             print(
-                f"TEST #{test_counter} SUCCEEDED in {execution_time_ms:.4f} milliseconds",
+                f"  TEST #{test_counter} SUCCEEDED in {execution_time_ms:.4f} milliseconds",
             )
```

Comment:
pretty unrelated, but this makes it easier to read the output of `python run_tests.py`